taqueria-plugin-taquito/transfer.ts

```diff
@@ -65,7 +65,7 @@ const configureToolKitWithSandbox = (parsedArgs: Opts, sandboxName: string): Pro
 	tezos.setProvider({
 		signer: new InMemorySigner((defaultAccount.secretKey as string).replace(/^unencrypted:/, '')),
 	});
-	return tezos;
+	return Promise.resolve(tezos);
```

Comment:
Why do we need this? The function is declared as `async` so isn't this essentially the same as `return tezos`

Reply:
Semantically, there's no difference here. However, `tsup` doesn't warn about an async without an await.

Reply:
What do you mean?

You mean to say "`tsup` does warn about an async without an await."?